db/migrations/20230125005019-create-proposals.js

```diff
@@ -0,0 +1,45 @@
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.createTable('Proposals', {
```

Comment:
Will the table be named `Proposals` in MySQL?

If so, maybe set `tableName` to `proposals`

Reply:
Done!